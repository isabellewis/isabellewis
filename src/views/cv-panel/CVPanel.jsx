import React from 'react';
import styled from 'styled-components';

import { SectionHeading, ShortCopy } from '../../elements';

const CVWrapper = styled.div`
  min-height: 100%;
  padding: 30px 0;
`;

export const CVPanel = () => (
  <CVWrapper id="cvPanel">
    <SectionHeading title="CV" />
    <ShortCopy shortCopyText="I have much experience" />
  </CVWrapper>
);
