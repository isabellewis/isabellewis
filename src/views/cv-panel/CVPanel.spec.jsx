import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
import { CVPanel } from './CVPanel';

describe('views/CVPanel', () => {
	let component;

	beforeEach(() => {
		component = shallow(<CVPanel />);
	});

	it('should match the snapshot', () => {
		expect(component).toMatchSnapshot();
	});
});