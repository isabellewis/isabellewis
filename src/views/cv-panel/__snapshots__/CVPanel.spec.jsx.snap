// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`views/CVPanel should match the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <CVPanel />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "children": Array [
        <SectionHeading
          title="CV"
        />,
        <ShortCopy
          shortCopyText="I have much experience"
        />,
      ],
      "id": "cvPanel",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "title": "CV",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "shortCopyText": "I have much experience",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
    ],
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": Array [
          <SectionHeading
            title="CV"
          />,
          <ShortCopy
            shortCopyText="I have much experience"
          />,
        ],
        "id": "cvPanel",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "title": "CV",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "shortCopyText": "I have much experience",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
